coderd/autobuild/schedule/schedule.go

```diff
@@ -108,6 +108,36 @@ func (s Schedule) Next(t time.Time) time.Time {
 	return s.sched.Next(t)
 }
 
+var t0 = time.Date(1970, 1, 1, 1, 1, 1, 0, time.UTC)
+var tMax = t0.Add(168 * time.Hour)
```

Comment:
Would this work correctly (or as expected) if e.g. max TTL is set to a different value in the database (i.e. the column that's added to `templates`)?

Reply:
We don't let people set the `month` or `day-of-month` field currently, so a schedule currently maxes out at 1 week.